jooq/src/main/kotlin/com/terraformation/backend/jooq/Config.kt

```diff
@@ -174,6 +175,8 @@ val ID_WRAPPERS =
             listOf(
                 IdWrapper("AutomationId", listOf("automations\\.id")),
                 IdWrapper("BalenaDeviceId", listOf("device_managers\\.balena_id")),
+                IdWrapper("CohortId", listOf("cohorts\\.id", ".*\\.cohort_id")),
+                IdWrapper("CohortPhaseId", listOf("cohort_phases\\.id", ".*\\.cohort_phase_id")),
```

Comment:
If the table is turned into an enum, you shouldn't need an ID wrapper class. I'm kind of surprised having both at the same time doesn't break the build!